autogen_subdomain.go

```diff
@@ -0,0 +1,58 @@
+package api
+
+import (
+	"gopkg.in/nullstone-io/go-api-client.v0/types"
+	"net/http"
+	"path"
+	"strconv"
+)
+
+type AutogenSubdomain struct {
+	Client *Client
+}
+
+// GET /orgs/:orgName/subdomains/:subdomainId/envs/:envName/autogen_subdomains
+func (a AutogenSubdomain) Get(subdomainId int, envName string) (*types.AutogenSubdomain, error) {
+	res, err := a.Client.Do(http.MethodGet, path.Join("subdomains", strconv.Itoa(subdomainId), "envs", envName, "autogen_subdomain"), nil, nil, nil)
```

Comment:
This says `autogen_subdomain` in the endpoint, but the func comment says `autogen_subdomains`.
Which one is right?

Reply:
Sorry, that was a typo, I have changed them all to autogen_subdomain since it is a singular resource.